tests/testthat/test-Rules-methods.R

```diff
@@ -1,5 +1,46 @@
 # nextBest ----
 
+## NextBestMTD ----
+
+test_that("nextBest-NextBestMTD returns correct next dose and plot", {
+  data <- h_get_data(placebo = FALSE)
+  model <- h_get_logistic_log_normal()
+  samples <- h_as_samples(
+    list(alpha0 = c(-2.38, -2.13, -1.43, -2.57), alpha1 = c(1.67, 1.3, 1.77, 2.51))
+  )
+  nb_mtd <- NextBestMTD(
+    target = 0.33,
+    derive = function(mtd_samples) {
+      quantile(mtd_samples, probs = 0.25)
+    }
+  )
+
+  result <- nextBest(nb_mtd, 90, samples, model, data)
+  expect_identical(result$value, 75)
+  vdiffr::expect_doppelganger("Plot of nextBest-NextBestMTD", result$plot)
```

Comment:
now looking at the plot - how about rotating the labels of the vertical lines 90 degrees such that they overlap less? just an idea to make it a bit better readable